src/Documentation/sidebar.json

```diff
@@ -48,6 +48,7 @@
     "name": "User Guide",
     "folder": "/static/docs/user-guide",
     "files": [
+      "glossary.md",
```

Comment:
The glossary page should probably be the last in this group.

Reply:
agreed with @jorgeorpinel , let's move it all the way down